includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
```

Comment:
@nimid agreed with above, except `If we have not plan to develop the translation or the translation feature will be not release soon. I agree to remain the text domain.`

like I said before, I'm not sure how long we gonna spend for another review and I also want to release this version without have to wait for another review.

So, if you're agree for `remain the text domain If we have not plan to develop the translation or the translation feature will be not release soon.` then, I'll wait for another commit and merge it asap.

thanks :)